assets/src/styles/blocks/core-overrides/Image.scss

```diff
@@ -34,6 +34,10 @@
     display: none !important;
   }
 
+  &.w-40 {
```

Comment:
Wouldn't we need to also fix the height to 40px? 🤔

Reply:
I did not add the height since we assume that the pictures there will be all square. 
However, following up your good suggestion I've replaced the `.w-40` with `.square-40` which is more consistent.

Reply:
Is there a downside to adding the height too? Especially if we're gonna rename it `square-40`, it seems it would make sense 🤔

Reply:
It's been added [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/5ba2ac61f944a9f42c399e1288d58ab1beb7951f/assets/src/styles/blocks/core-overrides/Image.scss#L17-L18). Does it make sense?

Reply:
Ah, yes, sorry 🤦‍♀️